paper/writeup.tex

```diff
@@ -238,32 +245,35 @@ \section{Introduction}
 recombination between distinct haplotypes can create a novel haplotype
 whose relationships and origination time could be inferred.
 
-Haplotype identity has been nearly overlooked in the literature on ARG inference so far --
-all metrics that have been used so far to measure accuracy of inferred ARGs
+Haplotype identity has been largely overlooked in the literature on ARG inference so far --
+in that the metrics that have been used to measure accuracy of inferred ARGs
 depend only on the sequence of marginal trees,
-not on sharing of ancestral haplotype identities between them.
+not on the sharing of ancestral haplotypes that span across these trees. % NSP-22NOV: what does "sharing of ancestral haplotype identities" mean, concretely? Would "... depend only on the sequence of marginal trees, but not on the ancestral haplotypes that span across these trees" be a clearer description?
+% HF-24NOV: Noted and changed
 Suppose two methods are used to infer ARGs from the same set of genotyped samples.
-Although both ARGs describes relationships between many haplotypes,
-only those haplotypes that represent the focal set of genotyped samples
+Although both ARGs describe the relationships between many haplotypes,
+only those haplotypes that represent the focal set of genotyped samples % NSP-22NOV: the meaning here isn't clear to me-- what does "represent the focal set of genotyped samples" mean, concretely?
+% HF-24NOV : I am not exactly sure, I will ping Peter
```

Comment:
I mean that if f() is a node-to-node mappings between ARGs, and `a` is a sample in the first ARG and `b` is the *same* sample in the second ARG, we want `f(a) = b`.